podaac/controllers/default_controller.py

```diff
@@ -0,0 +1,527 @@
+        data['error'] = f"404: Results with the specified Feature ID {feature_id} were not found."
+    elif len(results) > 5750000:
+        data['error'] = f'413: Query exceeds 6MB with {int(len(results.split(",")))} hits.'
+
+    else:
+        data['status'] = "200 OK"
+        data['time'] = str(time) + " ms."
+        #data['search on'] = {"featureID": feature_id}
+        data['type'] = "FeatureCollection"
+        data['features'] = []
+        i = 0
+        print(len(results))
+        for t in results:
+            if ((t['time'] != '-999999999999')): #and (t['width'] != '-999999999999')):
+                feature = {}
+                feature['properties'] = {}
+                feature['geometry'] = {}
+                feature['type'] = "Feature"
+                feature['geometry']['coordinates'] = []
+                type = ''
+                if 'POINT' in t['geometry']:
+                    geometry = t['geometry'].replace('POINT (', '').replace(')', '')
+                    geometry = geometry.replace('"', '')
+                    geometry = geometry.replace("'", "")
+                    type = 'Point'
+                if 'LINESTRING' in t['geometry']:
+                    geometry = t['geometry'].replace('"LINESTRING (', '').replace(')"', '')
+                    geometry = geometry.replace('"', '')
+                    geometry = geometry.replace("'", "")
+                    type = 'LineString'
+                feature['geometry']['type'] = type
+                print(geometry)
+                for p in geometry.split("; "):
+                    (x, y) = p.split(" ")
+                    if type == 'LineString':
+                        feature['geometry']['coordinates'].append([float(x), float(y)])
+                    if type == 'Point':
+                        feature['geometry']['coordinates'] = [float(x), float(y)]
+                i += 1
+                feature['properties']['time'] = datetime.fromtimestamp(float(t['time'])+946710000).strftime("%Y-%m-%d %H:%M:%S")
+                feature['properties']['feature_id'] = float(t['feature_id'])
+                feature['properties']['wse'] = float(t['wse'])
+                feature['properties']['slope'] = float(t['slope'])
+                data['features'].append(feature)
+
+        data['hits'] = i
+
+    return data
+
+def format_csv(cur, feature_id, exact, time, fields):
+    """
+
+    Parameters
+    ----------
+    cur
+    swot_id
+    exact
+    time
+
+    Returns
+    -------
+
+    """
+    # Fetch all results from mySQL query
+    results = cur.fetchall()
+
+    data = {}
+
+    print(fields)
+    if results is None:
+        data['error'] = f"404: Results with the specified Feature ID {feature_id} were not found."
+    elif len(results) > 5750000:
+        data['error'] = f'413: Query exceeds 6MB with {int(len(results.split(",")))} hits.'
+
+    else:
+        #csv = "feature_id, time_str, wse, geometry\n"
+        csv = fields + '\n'
```

Comment:
Python has a `csv` module that may make this easier as well https://docs.python.org/3/library/csv.html